changelog.d/20251125_162102_grigorio714_cloud_storage_failed_status_fix.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Fixed cloud storage status. Unavailable storages (S3, Azure) now return NOT_FOUND status instead of 400 Bad Request
```

Comment:
```suggestion
- Fixed cloud storage status. Unavailable storages now return NOT_FOUND status instead of 400 Bad Request
```

We've now adjusted all three storages, so there's no need to list them.